conf/config.go

```diff
@@ -21,6 +21,8 @@ var (
 	HTTPBind   = env.Get("HTTP_BIND", PublicIPv4.String()+":80")
 	BOOTPBind  = env.Get("BOOTP_BIND", PublicIPv4.String()+":67")
 
+	ScriptPatch = env.Get("IPXE_SCRIPT_PATCH", "")
```

Comment:
This is an old pattern that we're trying to move away from. _"Only func main has the right to decide which flags, env variables, config files are available to the user"_ -- [ref](https://github.com/tinkerbell/boots/blob/main/docs/DESIGNPHILOSOPHY.md#functionsmethods)

Would you mind adding this as a cli flag [here](https://github.com/tinkerbell/boots/blob/c92674c5f39f7005602bc6882afd035dc0f13d0c/cmd/boots/main.go#L359) instead? This will also make the cli flag name available as an env var (`BOOTS_IPXE_SCRIPT_PATCH`).